src/uu/split/src/split.rs

```diff
@@ -138,6 +150,13 @@ size is 1000, and default PREFIX is 'x'. With no INPUT, or when INPUT is
     settings.input = input;
     settings.prefix = prefix;
 
+    settings.filter = matches.opt_str("filter");
+
+    if settings.filter.is_some() && cfg!(windows) {
+        // see https://github.com/rust-lang/rust/issues/29494
+        crash!(1, "--filter is not supported in this platform")
```

Comment:
Would it be possible to use show_error and return instead of crash?

Reply:
will do

Reply:
And don't hesitate to open an issue on this project as good first bug for potential contributor to implement this on Windows too :)